### PR TITLE
Sponsors feat

### PR DESCRIPTION
**Pull request by gorajski**

### CODE REVIEW

**Review by j1mc — COMMENTED** [review]
The changes to the sponsor info pages are great. The "about" page seems a bit broken in terms of styling now, though. Company member images seem to overlap now.

File: _layouts/sponsor_show.html
Comment (TylerSmall19) [inline _layouts/sponsor_show.html:8]: These links break on my browser (current version of Chrome) so we will need a fallback option for people who can't open mailto links.

File: _layouts/default.html
Comment (TylerSmall19) [inline _layouts/default.html]: Why did all of this white-space get added?

File: _layouts/about.html
Comment (TylerSmall19) [inline _layouts/about.html]: These start `<h3>` but end `</h2>`?

File: _layouts/about.html
Comment (TylerSmall19) [inline _layouts/about.html]: `` tags don't need `</img>` tags.

File: assets/css/main.css
Comment (TylerSmall19) [inline assets/css/main.css]: This isn't resizing the images like it is probably intended to do, it's only being put on the `div` that stores the images, causing the over-flow to break the page.

Comment (TylerSmall19) [reply assets/css/main.css]: Putting this on the images themselves would likely fix the issue, but they're all different aspect ratios, so you'd have to set only the width or height to keep the image from getting distorted.

**Review by TylerSmall19 — CHANGES_REQUESTED** [review]
The sponsors look great, but there are a few changes needed for the about page before we can merge. I left comments with possible causes/fixes.

File: _layouts/about.html
Comment (TylerSmall19) [reply _layouts/about.html]: Also, I noticed a bug with the templating. If spaces exist in HTML for alt text, it needs to wrapped in strings. `{{staff.name}}` would fix the bug.

File: _layouts/sponsor_show.html
Comment (gorajski) [reply _layouts/sponsor_show.html:8]: I agree.  I had a fallback in the original work from Sunday that got lost.  I will reinsert and modify.  May not look CSS pretty at first, but I can revisit later.  Would rather not hold up the request.

File: _layouts/default.html
Comment (gorajski) [reply _layouts/default.html]: I don't know.  I was trying to follow your suggestion of 4 spaces for HTML, so I added extra space.  Not sure why it was interpreted as so much extra space.  Will attempt a fix.

File: _layouts/about.html
Comment (gorajski) [reply _layouts/about.html]: Your points are valid. It's not code that I touched, but will fix.

File: _layouts/about.html
Comment (gorajski) [reply _layouts/about.html]: My bad, will fix.

File: assets/css/main.css
Comment (gorajski) [reply assets/css/main.css]: This is a known issue that I will be working on in an upcoming pull request.  The intention of the current pull request is to address the sponsor's page, but because I had to splice things together to repair the merge issue, this might have slipped in there.  I'm going to leave this alone for now.  Look for fixes in next pull request.

File: _layouts/sponsor_show.html
Comment (gorajski) [reply _layouts/sponsor_show.html:8]: Updated.  Will style later.

File: assets/css/main.css
Comment (gorajski) [reply assets/css/main.css]: Ignored for now.  Look for changes in upcoming pull request.

File: _layouts/about.html
Comment (gorajski) [reply _layouts/about.html]: Fixed.

File: _layouts/about.html
Comment (gorajski) [reply _layouts/about.html]: Fixed.

File: _layouts/default.html
Comment (gorajski) [reply _layouts/default.html]: Fixed.

File: _layouts/about.html
Comment (TylerSmall19) [inline _layouts/about.html]: These still need to be wrapped in strings. Otherwise you see things like `tyler` where it thinks small is now a property of the image.

**Comment by hollabaq86** [comment]
omg tags should now show strings for the alt attribute, please let me know if there's anything else outstanding you see

**Comment by gorajski** [comment]
All of these changes have been merged already from a pull request I created after this request.  I am closing it as it is no longer needed.

**Pull request closed without merging**
